lambdas/data-migration1/src/errors.ts

```diff
@@ -1,3 +1,4 @@
 import { createErrorType } from '@cumulus/errors';
 
 export const RecordAlreadyMigrated = createErrorType('RecordAlreadyMigrated');
+export const ColumnDoesNotExist = createErrorType('ColumnDoesNotExist');
```

Comment:
I think this error type is no longer necessary/used

Reply:
Yes, that's correct, I'll remove it